build.xml

```diff
@@ -216,7 +216,7 @@
         </antcall>
     </target>
     
-    <target name="dist" depends="init"><!-- TODO does not work with depends="init" only works if init run before --> <!-- TODO if depends init works add to all other targets and introduce global initialized param so that init will only be reexecuted if init not run before -->
+    <target name="dist" depends="init, clean"><!-- TODO does not work with depends="init" only works if init run before --> <!-- TODO if depends init works add to all other targets and introduce global initialized param so that init will only be reexecuted if init not run before -->
```

Comment:
Is the TODO still valid? I'm not sure what it's trying to say...

Reply:
thx for spotting will double check

Reply:
@bwbohl Why  `clean` and not `reset` target (i.e., `clean` plus delete `/lib`) ? When `dist` depends on `init`, the `/lib` folder should be reinstantiated anyway, right? Pobably it would require to switch order of depends: `depends="reset,init"`

Reply:
`reset` would be an option if you are developing the build process, but I deem it kind of annoying when the lib components are pulled again just when I (locally) do some schema development. But yeah that points me in another direction why not introduce a `build` or `build-all` target for those local development cases (or general building) and make `dist` a target that puts all those build artefacts in the right places in the list folder; the primary function of `dist` then maybe should be our own build-and-deploy workflow?

Reply:
concerning the TODO describing problems with `init`:
It's not working to just call `dist` at the moment (without calling `init` before) because the build-rng target fails without calling ant with`-lib` giving the path to Saxon. but if you give the path to where Saxon will be after init the call will fail due to missing file, although it is there hen build-rng is called, it wasn't when the build was started…

Reply:
I have an idea o how to fix it though… ;-)

Reply:
Do you want to fix it in this PR, or open a new one later?